### PR TITLE
Replace deprecated ioutil calls with os equivalent

### PR DESCRIPTION
**Pull request by noelukwa**
# Description of the issue
This PR replaces calls to ioutil module which is deprecated with os module equivalent, it fixes this issue [648](https://github.com/aws/amazon-cloudwatch-agent/issues/648)

# Description of changes
_How does this change address the problem?_

# License
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

# Tests
_Describe what tests you have done._

# Requirements
_Before commit the code, please do the following steps._
1. Run `make fmt` and `make fmt-sh`
2. Run `make linter`

### CODE REVIEW

**Comment by noelukwa** [comment]
@SaxyPandaBear Please have a look at this.

File: cfg/migrate/awscsm.go [hunk 1]
```diff
@@ -9,10 +9,10 @@ func init() {
 	AddRule(AwsCsmRule)
 }
 
-//  [agent]
-//-   enable_csm_reflection = true
-//    [[inputs.awscsm_listener]]
-//+     data_format = "aws_csm"
+//		[agent]
```
Comment (SaxyPandaBear) [inline cfg/migrate/awscsm.go:12]: There are a lot of extraneous formatting changes, and not all of them are warranted, like this one. What did you run to format these files?

Comment (noelukwa) [reply cfg/migrate/awscsm.go:12]: I believe i did `Make fmt`

Comment (noelukwa) [reply cfg/migrate/awscsm.go]: Apologies, i just realised my editor ran auto format

**Review by khanhntd — APPROVED** [review]
Thanks for the contribution, Noel ! It's very good to see function got up to open source standard!

**Review by williazz — APPROVED** [review]
thanks for the contribution

**Pull request merged**